src/components/Header.tsx

```diff
@@ -33,18 +33,18 @@ export default function Header() {
           </div>
         </div>
 
-        <div className="gap-2 font-normal space-x-4 text-xs flex justify-end text-gray-500 p-2">
+        <div className="gap-2 font-normal items-center space-x-4 text-xs flex text-gray-500 p-2">
           <a href="/#">Help</a>
           <a href="/#">Find In-Store</a>
           {headerIcons.map((image, index) => (
-            <span className="flex justify-items-center">
+            <span className="flex" key={index}
+            >
```

Comment:
Is this eslint? How come the spacing is so strange?

Reply:
not certain, addressed.